# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/mozilla/meta/webgpu/cts/webgpu/shader/validation/expression/call/builtin/atan2/cts.https.html.ini
# Commit: 3f167f8b3137
# Full Hash: 3f167f8b313749f50f135e189e5854bef7ac0c73
# Author: Erich Gubler <erichdongubler@gmail.com>
# Date: 2025-10-03 18:08 +0000
# Regressor Bug: 1991551
# File Overlap Count: 23
# Overlapping Files: third_party/rust/wgpu-hal/Cargo.toml, supply-chain/audits.toml, third_party/rust/naga/src/back/spv/writer.rs, third_party/rust/naga/src/back/msl/writer.rs, third_party/rust/wgpu-types/src/lib.rs
# Description:
#   Bug 1991551 - build(webgpu): update WGPU to 8c4aebc0c1b75e3f7412dfc180745a0717fc30ec r=webgpu-reviewers,supply-chain-reviewers,nical
# ==============================================================================

diff -r 823439f307a0 -r 3f167f8b3137 testing/web-platform/mozilla/meta/webgpu/cts/webgpu/shader/validation/expression/call/builtin/atan2/cts.https.html.ini
--- a/testing/web-platform/mozilla/meta/webgpu/cts/webgpu/shader/validation/expression/call/builtin/atan2/cts.https.html.ini	Fri Oct 03 18:01:30 2025 +0000
+++ b/testing/web-platform/mozilla/meta/webgpu/cts/webgpu/shader/validation/expression/call/builtin/atan2/cts.https.html.ini	Fri Oct 03 18:08:29 2025 +0000
@@ -1,9 +1,7 @@
 [cts.https.html?q=webgpu:shader,validation,expression,call,builtin,atan2:invalid_argument_x:*]
-  implementation-status: backlog
   [:type="bool"]
 
   [:type="f32"]
-    expected: FAIL
 
   [:type="i32"]
 
@@ -65,11 +63,9 @@
 
 
 [cts.https.html?q=webgpu:shader,validation,expression,call,builtin,atan2:invalid_argument_y:*]
-  implementation-status: backlog
   [:type="bool"]
 
   [:type="f32"]
-    expected: FAIL
 
   [:type="i32"]
 
@@ -136,26 +132,21 @@
     expected: FAIL
 
   [:use=true]
-    expected: FAIL
 
 
 [cts.https.html?q=webgpu:shader,validation,expression,call,builtin,atan2:parameters:*]
   implementation-status: backlog
   [:test="af"]
-    expected: FAIL
 
   [:test="af_ai"]
-    expected: FAIL
 
   [:test="af_f16"]
 
   [:test="af_f32"]
 
   [:test="ai"]
-    expected: FAIL
 
   [:test="ai_af"]
-    expected: FAIL
 
   [:test="ai_f16"]
 
@@ -221,63 +212,42 @@
 
 
 [cts.https.html?q=webgpu:shader,validation,expression,call,builtin,atan2:values:*]
-  implementation-status: backlog
   [:stage="constant";type="abstract-float"]
-    expected: FAIL
 
   [:stage="constant";type="f16"]
-    expected: FAIL
 
   [:stage="constant";type="f32"]
-    expected: FAIL
 
   [:stage="constant";type="vec2%3Cabstract-float%3E"]
-    expected: FAIL
 
   [:stage="constant";type="vec2%3Cf16%3E"]
-    expected: FAIL
 
   [:stage="constant";type="vec2%3Cf32%3E"]
-    expected: FAIL
 
   [:stage="constant";type="vec3%3Cabstract-float%3E"]
-    expected: FAIL
 
   [:stage="constant";type="vec3%3Cf16%3E"]
-    expected: FAIL
 
   [:stage="constant";type="vec3%3Cf32%3E"]
-    expected: FAIL
 
   [:stage="constant";type="vec4%3Cabstract-float%3E"]
-    expected: FAIL
 
   [:stage="constant";type="vec4%3Cf16%3E"]
-    expected: FAIL
 
   [:stage="constant";type="vec4%3Cf32%3E"]
-    expected: FAIL
 
   [:stage="override";type="f16"]
-    expected: FAIL
 
   [:stage="override";type="f32"]
-    expected: FAIL
 
   [:stage="override";type="vec2%3Cf16%3E"]
-    expected: FAIL
 
   [:stage="override";type="vec2%3Cf32%3E"]
-    expected: FAIL
 
   [:stage="override";type="vec3%3Cf16%3E"]
-    expected: FAIL
 
   [:stage="override";type="vec3%3Cf32%3E"]
-    expected: FAIL
 
   [:stage="override";type="vec4%3Cf16%3E"]
-    expected: FAIL
 
   [:stage="override";type="vec4%3Cf32%3E"]
-    expected: FAIL